### DISCUSSION TITLE
Can I use both oh-my-zsh and zinit ?

### DISCUSSION THREAD

**Discussion by NamekMaster** [zdharma-continuum/zinit#377] [Q&A] [answered]
I' using oh-my-zsh with antigen now, since antigen don't update for years, I think it's time to find another plugin manager. I'd like to keep oh-my-zsh, but it lack of management of external themes and plugins. Here I found zinit for plugins management. Is it ok if I keep oh-my-zsh and just add zinit at end of .zshrc?
It tried and it seems ok, but I don't known whether there are any potential problems.

**Comment by psprint** [comment, marked as answer]
There will be no problems, although you could load OMZ plugins via zinit
and skip using full OMZ separately.

**Reply by NamekMaster** [reply]
Do you mean in this way if I load OMZ built-in plugins via zinit that is equivalant to OMZ?

**Reply by 3v1n0** [reply]
You can load also some core OMZ features if you want, i've quite complex config files that can show how I'm dealing with it:

https://github.com/3v1n0/ellipses/blob/chezmoi/private_dot_ellipses/shells/private_dot_3v1rc#L804-L819

Also see [this function](https://github.com/3v1n0/ellipses/blob/chezmoi/private_dot_ellipses/shells/private_dot_3v1rc#L730-L784) to see how `$_my_omz_plugins` can be loaded once `$ZSH` is defined.

**Comment by psprint** [comment]
Yes, no difference
